.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -4,9 +4,13 @@ about: Create a report to help us improve
 title: ''
 labels: bug
 assignees: ''
-
 ---
 
+**Have you**
+
+- [ ] Read [Troubleshooting Guide](https://azure.github.io/secrets-store-csi-driver-provider-azure/troubleshooting/)
+- [ ] Searched on [GitHub issues](https://github.com/Azure/secrets-store-csi-driver-provider-azure/issues)
```

Comment:
do we want to add [discussions](https://github.com/Azure/secrets-store-csi-driver-provider-azure/discussions)?

Reply:
Thanks for the suggestion. Added it!